solutions/platinum/leetcode-1453.mdx

```diff
@@ -0,0 +1,106 @@
+---
+id: leetcode-1453
+source: Leetcode
+title: Maximum Number of Darts Inside of a Circular Dartboard
+author: Mihnea Brebenel
+---
+
+## Explanation
+
+The general idea is the following: place each point on the circle boundary,
+then sweep - or rotate - the circle around the point to see the maximum
+number of points that can be inside the circle. The picture below depicts
+how the sweep works:
```

Comment:
what point? the points given in the input? how do we know where to place this initial circle?
> around the point
"the" point? you mentioned placing multiple points.

Reply:
I don't mean all points at once, but for each one try it and place it on the circle's boundary.

Reply:
I rephrased it